src/components/AccountSelector/AccountSelectButton/AccountSelector.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { FiUser } from "react-icons/fi";
+import { MassaToken } from "../../../assets/svg-components/MassaToken";
+import { AccountSelectorButton } from "./AccountSelectorButton";
+
+export default {
+  title: "Components/Account Selector",
+  component: AccountSelectorButton,
+};
+
+const theme = "theme-light";
+
+const args = {
+  profileimage: <FiUser className="text-neutral h-6 w-6" />,
+  accountname: "account #",
+  svg: <MassaToken size={"24"} />,
```

Comment:
If size is in pixel, you can replace by :

```suggestion
  svg: <MassaToken size={24} />,
```

Reply:
Yes !